Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hss/fha/nhinc/admingui/servlet/InitServlet.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2009-2016, United States Government, as represented by the Secretary of Health and Human Services.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above
+ *       copyright notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the documentation
+ *       and/or other materials provided with the distribution.
+ *     * Neither the name of the United States Government nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package gov.hss.fha.nhinc.admingui.servlet;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gov.hhs.fha.nhinc.admingui.display.DirectDisplayController;
+import gov.hhs.fha.nhinc.admingui.display.FhirDisplayController;
+
+/**
+ * Started on webapplication init, checks whether to enable/disable displays during the initial login 
+ * to the ADMIN GUI.
+ */
+public class InitServlet extends HttpServlet {
+
+	private static final long serialVersionUID = -7194100487044821581L;
+	private static final Logger LOG = LoggerFactory.getLogger(InitServlet.class);
+
+	@Override
+    public void init(ServletConfig config) throws ServletException {
+		super.init(config);
+		
+		//The below checkDisplay() calls were originally called in a checkDisplay method in the LoginBean.
```

Comment:
We can find this information out via commit history -- it would be better to have comments in the code to explain why it's there, instead of why it's not somewhere else.